collector/pg_extensions.go

```diff
@@ -0,0 +1,112 @@
+package collector
+
+import (
+	"context"
+	"database/sql"
+	"github.com/go-kit/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"strconv"
+)
+
+func init() {
+	registerCollector("extensions", defaultEnabled, NewExtensionsCollector)
```

Comment:
can we avoid using init?

Reply:
We can but it would require refactoring of other subcollectors of 'postgres' collector
other sub-collectors of postgres_exporters use it here as well.

I would like to not to do that in the scope of this ticket to avoid possible risks.

wdyt?

Reply:
sure, no problem